src/test_workflow/smoke_test/smoke_tests_spec/default/security.yml

```diff
@@ -0,0 +1,90 @@
+---
```

Comment:
Where can I find the cluster configuration for the cluster that all of these tests are run against?

I think these APIs are a good start and cover the core APIs of the security plugin. If we want to get more exhaustive with test coverage we can either try to derive tests from the open-api-specification repo or cover the APIs outlined on https://opensearch.org/docs/latest/security/access-control/api/ (note: some of these APIs require additional `opensearch.yml` settings in order to call and are disabled by default like the config update API)

Reply:
The smoke tests are intended to be light weighted so it runs on default configuration without any customized cluster configuration at this time.

Reply:
Hey @cwperks do you have any other suggestions on these API test cases for security plugins? Let me know if there is anything else you want me to onboard. Thanks!

Reply:
This covers some of the core APIs so I think this is a good base to start with.